### PR TITLE
Setup base for neetoarea51

### PR DESCRIPTION
**Pull request by Anamika-123**

### CODE REVIEW

**Comment by Anamika-123** [comment]
@chiraggshah _a Could you please review this PR?

File: app/javascript/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import Main from "./components/Main";
+
+const App = () => {
+  return (<Main/>);
+};
```
Comment (chiraggshah) [inline app/javascript/src/App.jsx:7]: use implicit return
```js
const App = () => (<Main/>);
```

File: app/javascript/src/components/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import { PageLoader } from "neetoui";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { ToastContainer } from "react-toastify";;
+
+import { ADMIN_URL } from "common/routes";
+import NeetoArea51 from "./NeetoArea51";
+
+const Main = () => {
+  const [loading, setLoading] = useState(true);
```
Comment (chiraggshah) [inline app/javascript/src/components/Main.jsx:11]: Remove if `loading` is not being used anywhere.

File: app/javascript/src/components/NeetoArea51/index.jsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+          to="/"
+          icon={LeftArrow}
+          iconPosition="left"
+          label="Back"
+          style="text"
+          className="flex-shrink-0 mr-5"
+        />
+        <Tab noUnderline>
+          {names.map((name, index) => {
+            return (
```
Comment (chiraggshah) [inline app/javascript/src/components/NeetoArea51/index.jsx:73]: Use implicit return

File: app/javascript/src/components/NeetoArea51/index.jsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+              < Tab.Item
+                key={index}
+                className="py-6 font-semibold tracking-wide uppercase"
+                onClick={() => {
+                  setModelName(name);
+                  setPageIndex(1);
+                  return loadRowData(name);
```
Comment (chiraggshah) [inline app/javascript/src/components/NeetoArea51/index.jsx:80]: Is return needed here?

**Review by chiraggshah — APPROVED** [review]
Added a few minor comments. Rest looks good.

File: app/javascript/src/App.jsx (hunk 1 above)
Comment (Anamika-123) [reply app/javascript/src/App.jsx]: Done.

**Pull request merged**